### ISSUE TITLE
`CalendarDatePicker` focus issues

### ISSUE THREAD

**Issue by jsinsel-kahua** [unoplatform/uno#7784]
### Current behavior

Create a new project and add a CalenderDatePicker control to the view.  The following focus issues were found:

1. When the flyout opens, focus is not set inside so you can't tab or navigate the calendar.

You can get focus to occur by mouse down on a date, moving off and then releasing the mouse button.  However, the following issues were present:
1. Left/right navigation does not work, only up/down.
2. If you tab around through the header and back to the dates, the wrong date is focused first.  It will sometimes show two dates with the focus indicator.
3. It will sometimes lose focus in the flyout altogether and you are unable to return focus to the flyout.
4. Using spacebar/enter does not always select a date.

### Expected behavior

Works like UWP.

### How to reproduce it (as minimally and precisely as possible)

Attaching a sample sln.
[App1.Shared.zip](https://github.com/unoplatform/uno/files/7839828/App1.Shared.zip)

### Works on UWP/WinUI

Yes

### Environment

Uno.UI / Uno.UI.WebAssembly / Uno.UI.Skia

### NuGet package version(s)

```
<PackageReference Include="Uno.Material" Version="1.0.3" />
<PackageReference Include="Uno.Microsoft.Toolkit.Uwp.UI.Controls" Version="7.1.0-build.45.gd2e4192c8c" />
<PackageReference Include="Uno.Microsoft.Toolkit.Uwp.UI.Controls.DataGrid" Version="7.1.0-build.45.gd2e4192c8c" />
<PackageReference Include="Uno.Microsoft.Xaml.Behaviors.Uwp.Managed" Version="2.0.3-uno.95" />
<PackageReference Include="Uno.UI.WebAssembly" Version="3.11.6" />
<PackageReference Include="Uno.UI.RemoteControl" Version="3.11.6" Condition="'$(Configuration)'=='Debug'" />
<PackageReference Include="Uno.Wasm.Bootstrap" Version="3.0.0" />
<PackageReference Include="Uno.Wasm.Bootstrap.DevServer" Version="3.0.0" />
```

### Affected platforms

WebAssembly

### IDE

Visual Studio 2022

### IDE version

Version 17.1.0 Preview 1.0

## Notes for contributors

This is unfortunately quite complex problem, as it involves implementing focus management handling from an internal underlying virtualized panel which is also used by `ListView`.

**Comment by darenm** [comment]
@MartinZikmund Any update on this one?

**Comment by darenm** [comment]
@jeromelaban @MartinZikmund Any update on this?

**Comment by jeromelaban** [comment]
We do not have updates on this topic at this time.

**Comment by ChaseKopp6** [comment]
issue still present

**Comment by MartinZikmund** [comment]
# Repro generator

This issue was classified as not reproducible with a code sample or test.

**Reason:** CalendarDatePicker focus/accessibility issue — requires complex focus/keyboard interaction testing.

**Comment by agneszitte** [comment]
@darenm, @ChaseKopp6 are you able to reproduce this one lately ?

**Comment by darenm** [comment]
@Agnes - close it.
